hivemind/__init__.py

```diff
@@ -3,4 +3,4 @@
 from hivemind.server import *
 from hivemind.utils import *
 
-__version__ = '0.8.2'
+__version__ = '0.8.3dev1'
```

Comment:
let's call all non-released branches as such, whaddaya say?

Reply:
(discussed internally — going with option 2, versioning remains as is)

Reply:
Agreed